.deploy/init.sh

```diff
@@ -2,36 +2,18 @@
 
 set -ex
 
-if [ ! -e /docker_build ]; then
+if ! [ -e /.docker_build ]; then
   echo "This script should only be ran from a Dockerfile"
   exit 1
 fi
```

Comment:
`.dockerenv` does not exist on buildkit but the cgroup approach should work.


See https://stackoverflow.com/a/69860299/955091